### PR TITLE
Task 2: first group of subtasks + modules + lazyload

### PR DESCRIPTION
**Pull request by lmhs**
Completed first group of subtasks.
 [Result](http://lmhs.github.io/FrontCamp/)
#7

### CODE REVIEW

**Comment by lmhs** [comment]
Completed additional group of subtasks:
modules + dynamic loading

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Repository containing tasks for FrontCamp",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "watch": "webpack --watch",
+    "dev": "NODE_ENV=development webpack-dev-server --open",
```
Comment (st-zelenin) [inline package.json]: do you work on Mac? I get compile error on windows... looks like due to this [issue](https://github.com/davezuko/react-redux-starter-kit/issues/162). I have to remove NODE_ENV=development to run on my machine.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "description": "Repository containing tasks for FrontCamp",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "watch": "webpack --watch",
```
Comment (st-zelenin) [inline package.json]: looks like we do not need to run separate 'watch' server as webpack-dev-server is able to do this. I have tested on my machine - everything is updated successfully with single 'npm run dev'

File: README.md [hunk 3]
```diff
@@ -8,7 +8,24 @@
...
 npm install
 ```
-3. Run Babel-cli
+
+### Project run:
+Run webpack
+
+- for development:
+```
+npm run watch
```
Comment (st-zelenin) [inline README.md]: looks like we do not need this

File: browserslist [hunk 4]
```diff
@@ -0,0 +1,2 @@
+last 2 version
```
Comment (st-zelenin) [inline browserslist:1]: is this file required by autoprefixer?

File: src/scripts/constants.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+// let and const
+// settings for desctructuring assignment example
+const settings = {
+  ['newsAPIURL'] : 'https://newsapi.org/v2'
+};
+
+let apiKey = 'bdbeeb170f8c47a2b97aa0f6252bfb90';
```
Comment (st-zelenin) [inline src/scripts/constants.js:7]: do we still need NewsRequest? it has the same apiKey declaration inside.

**Review by st-zelenin — COMMENTED** [review]
very good. some minor questions.

File: README.md (hunk 3 above)
Comment (lmhs) [reply README.md]: yeah, removed

File: browserslist (hunk 4 above)
Comment (lmhs) [reply browserslist:1]: yes, autoprefixer needs a list of supported browsers.
as an enhancement next step would be find a solution that lets babel-preset-env use the browserslist file

File: package.json (hunk 2 above)
Comment (lmhs) [reply package.json]: yeah, removed

File: package.json (hunk 1 above)
Comment (lmhs) [reply package.json]: yes, I do
ok, I will check on windows machine

File: src/scripts/constants.js (hunk 5 above)
Comment (lmhs) [reply src/scripts/constants.js:7]: No, I will remove the file as it is not transpiling correctly
